src/transformers/modeling_encoder_decoder.py

```diff
@@ -110,8 +110,11 @@
             If set to :obj:`True`, :obj:`past_key_values` key value states are returned and can be used to speed up
             decoding (see :obj:`past_key_values`).
         output_attentions (:obj:`bool`, `optional`):
-            Whether or not to return the attentions tensors of all attention layers. See ``attentions`` under returned
-            tensors for more detail.
+            Whether or not to return the attentions tensors of all encoder attention layers. See ``attentions`` under
+            returned tensors for more detail.
+        decoder_output_attentions (:obj:`bool`, `optional`):
```

Comment:
I don't think we need this. If the user sets `output_attentions=True`, he will get all available output attentions.

Reply:
I discovered by experimentations that when setting `output_attentions=true`, only the encoder attentions are returned

Reply:
And then i noticed that this was subsequently fixed, I've removed